include/benchmark/reporter.h

```diff
@@ -155,7 +156,7 @@ class BenchmarkReporter {
 // Simple reporter that outputs benchmark data to the console. This is the
 // default reporter used by RunSpecifiedBenchmarks().
 class ConsoleReporter : public BenchmarkReporter {
- public:
+public:
```

Comment:
Hello dominic. I removed the space because JSONReporter and CSVReporter classes had no space before "public:". So I though that we either add an space on those other to classes or remove the space in this one, to keep consistency. Is there a preferred style?